src/components/cms/MyDevices/MapContainer.js

```diff
@@ -115,13 +132,17 @@ class MapContainer extends Component {
             visible={showingInfoWindow}
           >
             <div>
-              <p>
-                {`Mac Address: ${selectedPlace.macId}`}
-                <br />
-                {`Room: ${selectedPlace.room}`}
-                <br />
-                {`Device Name: ${selectedPlace.title}`}
-              </p>
+              {tooltipRender !== 'contactMap' ? (
```

Comment:
This would then be `this.props.tooltipRenderer(data)`

Reply:
Done  Sir.